src/Action.js

```diff
@@ -0,0 +1,17 @@
+import React, {PropTypes} from 'react';
+
+const Action = props => {
```

Comment:
I would be careful with the naming here. Action usually refers to an actual action. So maybe `ActionHistoryListItem` would be more descriptive.